client/src/pages/HouseDetails/index.js

```diff
@@ -36,11 +48,20 @@ const HouseDetails = () => {
             return <Housemate key={user.user}>{user.user}</Housemate>
           })}
         </HouseMates>
-        <Button
-          onClick={setHouseDetail}
-          userHouseNames={houseNames}
-          houseDetail={houseDetail}
-        />
+        <TicketButtonContainer>
+          <Button
+            onClick={setHouseDetail}
+            userHouseNames={houseNames}
+            houseDetail={houseDetail}
+            setCurrentHouse={set}
```

Comment:
Dont' use `set` if we're clearing the item from local storage. There is a remove method https://github.com/seanyboy49/movie-night-fight/blob/76fb67ddcad2257e542b4385b071f5945d021eae/client/src/hooks/useLocalStorage/index.js#L24